processor/file.go

```diff
@@ -76,7 +76,7 @@ func walkDirectoryParallel(root string, output chan *FileJob) {
 
 	var wg sync.WaitGroup
 
-	var isSoloFile bool = false
+	var isSoloFile = false
```

Comment:
If we're cleaning this up, either just `var isSoloFile bool` (`false` is the zero value for bools) or `isSoloFile := false` would be my preference.

Reply:
Ah good point. I just did the refactor thing in the IDE and it picked that up. Not sure why it didn't go the whole way though.

Reply:
Done.